modules/op-tkn-trigger-management.adoc

```diff
@@ -75,3 +96,33 @@ List TriggerTemplates.
 ----
 $ tkn triggertemplate list -n myspace
 ----
+== clustertriggerbinding
+Manage ClusterTriggerBindings.
+
+.Example: Display ClusterTriggerBindings help
+----
+$ tkn clustertriggerbinding -h
+----
+
+== clustertriggerbinding delete
+Delete a ClusterTriggerBinding.
+
+.Example: Delete `myclusterbinding1` and `myclusterbinding2` ClusterTriggerBindings
+----
+$ tkn clustertriggerbinding delete myclusterbinding1 myclusterbinding2
+----
+== clustertriggerbinding describe
+Describe a ClusterTriggerBinding.
+
+.Example: Describe the `myclusterbinding` ClusterTriggerBinding
```

Comment:
The title says myclusterbinding but the one actually being described in the example is called mybinding

Reply:
done